Dockerfile

```diff
@@ -135,14 +148,12 @@ RUN luarocks install luacheck
 # NPM setup
 # FIXME: we should use package.json from coala
 RUN npm install -g \
-  alex \
```

Comment:
@jayvdb Are you aware of the conflicting package name in the haskell compiler (ghc i think) called alex ? We've had issues before where the binary used is the wrong binary ...